lib/output-files.js

```diff
@@ -24,8 +25,19 @@ class OutputFiles {
 
     var str = ``
 
-    // Get the last element in the path name
-    var truncatedPath = pathLib.basename(path)
+    var urlPath = new url.URL(path)
+    var postProtocolPath = urlPath.pathname.substring(1)
+
+    if (urlPath.hostname) {
```

Comment:
I like this approach for detecting whether not the path is URL; hooray for the new `URL` class.

I think we should pull this logic into a helper method, perhaps something along the lines of `handleURLPath`.

Reply:
Done with a tweak: extracted this into a new helper method that parses all paths using the URL class, even for paths missing the protocol.